src/dataconnect/loadAll.ts

```diff
@@ -0,0 +1,7 @@
+import { load, readFirebaseJson } from "./load";
+import type { Config } from "../config";
+
+export function loadAll(projectId: string, config: Config) {
```

Comment:
Since your PR, I added `loadAll` in a different file~

https://github.com/firebase/firebase-tools/blob/2f0b457b299019ada540f0ebd96c64070af19578/src/dataconnect/load.ts#L61C23-L61C30